### PR TITLE
add show text index.php to easy mode before not show link

### PR DESCRIPTION
**Pull request by jirawat050**
before 
screen shot 2017-11-07 at 1 11 29 pm
new 
screen shot 2017-11-07 at 1 46 25 pm

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -140,9 +140,9 @@
             <div id="test_box-container">
                 <ul class="ui-tabs-nav">
                     <li class="analytical_review ui-state-default ui-corner-top ui-tabs-selected ui-state-active"><a href="#">Advanced Testing</a></li>
-                    <li class="easy_mode"><a href=\"/easy.php\">Simple Testing</a></li>
-                    <li class="visual_comparison"><a href="/video/">Visual Comparison</a></li>
-                    <li class="traceroute"><a href="/traceroute.php">Traceroute</a></li>
+                    <li class="easy_mode"><a href="easy.php">Simple Testing</a></li>
```
Comment (pmeenan) [inline www/index.php]: WPT doesn't support relative paths.  You need to keep the absolute paths in place.

Comment (jirawat050) [reply www/index.php]: if correct edit to a href ="/easymode" ?

Comment (jirawat050) [reply www/index.php]: if is echo not show text

Comment (pmeenan) [reply www/index.php]: no, it is "/easy.php" or "/easy".  There is no /easymode rewrite rule

**Comment by jirawat050** [comment]
Okai I edit finish thankyou

File: www/index.php (hunk 1 above)
Comment (jirawat050) [reply www/index.php]: ok i edit finish thank you

Comment (jirawat050) [reply www/index.php]: Can PR complete the merge.

**Comment by jirawat050** [comment]
@pmeenan  why edit correct?if not correct because?

**Comment by pmeenan** [comment]
It is not correct because easy mode needs the profiles file to work which is why it is inside the if block:

/settings/profiles.ini

**Pull request closed without merging**